packages/terra-overlay/translations/de.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Where did we get this translations? Does the translations team need to review these?